spec/models/ability_spec.rb

```diff
@@ -180,8 +195,10 @@
         should be_able_to(:destroy, my_venue)
       end
 
-      it{ should be_able_to(:new, Conference) }
-      it{ should be_able_to(:create, Conference) }
+      it{ should_not be_able_to(:new, Organization)}
+      it{ should_not be_able_to(:create, Organization)}
+      it{ should_not be_able_to(:new, Conference)}
+      it{ should_not be_able_to(:create, Conference) }
```

Comment:
I think I figured out why these tests are failing.
From the [CanCanCan wiki](https://github.com/CanCanCommunity/cancancan/wiki/checking-abilities#checking-with-class):

> Important: If a block or hash of conditions exist they will be ignored when checking on a class, and it will return true. For example:
>
> ```ruby
> can :read, Project, :priority => 3
> can? :read, Project # returns true
> ```

Since the ability for Organizers is defined as

```ruby
can :manage, Conference, id: conf_ids
```

then

```ruby
can? :manage, Conference
```

will return `true`.
That's why the test **should** use `.new` in this case.